server/app.js

```diff
@@ -25,6 +25,9 @@ require('./config/express')(app, authenticate);
 // Link routes
 require('./config/routes').initialise(app);
 
+// Load and configure plugins
+require('../plugins/pluginManager')(app, config);
```

Comment:
I agree and will clean this up soon.  We do need some way that the plugins have access to the db connector, application name, and other environmental variables, but that could be defined in their own configuration files.